src/Microsoft.Health.Fhir.SqlServer.Api/Registration/FhirServerBuilderSqlServerRegistrationExtensions.cs

```diff
@@ -124,6 +129,21 @@ public static IFhirServerBuilder AddExperimentalSqlServer(this IFhirServerBuilde
             return fhirServerBuilder;
         }
 
+        /// <summary>
+        /// Adds the Schema worker background service.
+        /// </summary>
+        /// <param name="fhirServerBuilder">The FHIR server builder.</param>
+        /// <returns>The builder.</returns>
+        public static IFhirServerBuilder AddSchemaWorker(
+            this IFhirServerBuilder fhirServerBuilder)
+        {
+            EnsureArg.IsNotNull(fhirServerBuilder, nameof(fhirServerBuilder));
+
+            fhirServerBuilder.Services.AddHostedService<SchemaJobWorkerBackgroundService>();
```

Comment:
Any downside in adding this directly into the `AddExperimentalSqlServer` method?

Reply:
Makes sense, Added directly into the AddExperimentalSqlServer method.